crates/aptos-bitvec/src/lib.rs

```diff
@@ -15,9 +15,9 @@ use std::{
     ops::{BitAnd, BitOr},
 };
 
-// Every u8 is used as a bucket of 8 bits. Total max buckets = 256 / 8 = 32.
+// Every u8 is used as a bucket of 8 bits. Total max buckets = 65536 / 8 = 8196.
```

Comment:
u32::max makes the potential vector too large, I don't know why there's originally a limit and it checks on the deserialization

Reply:
I think we can loose the `MAX_BUCKETS` - there shouldn't be any need to put this limit in place.

Reply:
agree the limit makes very little sense, I'll remove it. u16::max is also a practical limit, there're too many things would break if we hit that number, also I've added a check on move side.

Reply:
now I understand why this needs to be checked at deserialization, a longer vector will break some api. for example the last_set_bit returns a u16 that can be overflowed by a vector that exceeds the limit